<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingGui::DlgSettingsGeneral</class>
 <widget class="QWidget" name="DrawingGui::DlgSettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>541</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="Templates">
     <property name="title">
      <string>Drawing Templates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>205</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Size of recent template list</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Gui::PrefSpinBox" name="recentTemplates">
            <property name="value">
             <number>4</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>Recent templates</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/Drawing</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Path for user templates</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="templatePath"/>
          </item>
          <item>
           <widget class="QToolButton" name="btnPath">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="OrthoProj">
     <property name="title">
      <string>Orthographic Projection Tool</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>205</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Minimum spacing between views (mm)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Gui::PrefSpinBox" name="orthoSpace">
            <property name="value">
             <number>4</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>Minimum space</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/Drawing</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Default projection</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Gui::PrefComboBox" name="defaultProj">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>Default projection</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/Drawing</cstring>
            </property>
            <item>
             <property name="text">
              <string>1st angle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3rd angle</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="Gui::PrefComboBox" name="comboTest">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="prefEntry" stdset="0">
       <cstring>comboTest</cstring>
      </property>
      <property name="prefPath" stdset="0">
       <cstring>Mod/Drawing</cstring>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefSpinBox</class>
   <extends>QSpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
